.github/workflows/deploy.yml

```diff
@@ -22,6 +22,9 @@ jobs:
         id: deploy
         uses: ./.github/actions/deploy-to-github-pages
       - name: Purge Cloudflare cache
+        env:
+          CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
+        if: ${{ env.CLOUDFLARE_TOKEN != '' }}
```

Comment:
Does this work without the curlies and the dollar sign?

Reply:
For the record, this required a small fixup to work: https://github.com/git/git-scm.com/commit/040e1877a28b5b5ae9e4b8a1abc194af70fc9430.

Reply:
🤦 thanks for catching that and fixing it! I appreciate it.

Reply:
@ttaylorr You're welcome! FWIW after running into similar troubles for one too many times, I made it a habit to cross-check my edits to GitHub workflows via https://rhysd.github.io/actionlint/. It does not catch 100% of my mistakes before "hitting production", but a sizable fraction, still.